packages/web/src/pages/api/game.ts

```diff
@@ -0,0 +1,7 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+import { d6, d20 } from '@adventure-bot/game/.build/src/utils/dice'
```

Comment:
These imports will be tightened up in a later PR. Eventually, you will be able to import from `game/utils/dice`, however, the first-class APIs will be exported from the root of the package.